pkg/types/installconfig.go

```diff
@@ -383,3 +389,20 @@ func (c *InstallConfig) WorkerMachinePool() *MachinePool {
 
 	return nil
 }
+
+// Capabilities defines the configuration for user to install/exclude optional OpenShift components
+type Capabilities struct {
```

Comment:
Is it useful to declare your own type?  I'd have expected you to use `*configv1.ClusterVersionCapabilitiesSpec` as the type for the new `InstallConfig` property.

Reply:
+1. Will remove. I was working myself backwards into reusing all the defs from configv1 but missed this one.

Reply:
The reason that we want to use our own type is because it allows us to control changes to meet our forward-compatibility guarantees.

Reply:
Won't openshift/api have those same forward-compat commitments?